pkg/services/store/object/sqlstash/sql_storage_server.go

```diff
@@ -273,6 +277,11 @@ func (s *sqlObjectServer) BatchRead(ctx context.Context, b *object.BatchReadObje
 }
 
 func (s *sqlObjectServer) Write(ctx context.Context, r *object.WriteObjectRequest) (*object.WriteObjectResponse, error) {
+	return s.AdminWrite(ctx, object.ToAdminWriteObjectRequest(r))
+}
+
+//nolint:gocyclo
+func (s *sqlObjectServer) AdminWrite(ctx context.Context, r *object.AdminWriteObjectRequest) (*object.WriteObjectResponse, error) {
```

Comment:
Can we fix lint issue here?

Reply:
not easily 😬 -- "gocyclo" essentially says the function is too complicated

We can, but it would mean moving the sequential bits inside the transaction into their own functions, but pass along the variables defined at the top